server/static/styles.css

```diff
@@ -15,7 +15,7 @@ body {
     justify-content: flex-start;
     align-items: center;
     width: 80%;
-    background-color: #ffffff;
+    background-color: "white";
```

Comment:
Didn't you mean "privilieged"?

Reply:
i can see where Igor got his humor from :)